app/templates/legal.hbs

```diff
@@ -36,7 +36,11 @@
   <p>
     The graphs, diagrams and charts in the <a href="/guides">Ember Guides</a> have been released under the Creative Commons Attribution-NonCommercial-ShareAlike (CC BY-NC-SA) license.
   </p>
-  <img class="pull-right" src="/images/legal/license-cc_by_nc_sa.png">
+  <img
+    alt="Legal license"
```

Comment:
This should probably have the details of the licence in the alt text (CC, By, Non Commercial, Share Alike)

Reply:
```suggestion
     alt="CC, By, Non Commercial, Share Alike"
```